.github/ISSUE_TEMPLATE/---bug.yml

```diff
@@ -0,0 +1,103 @@
+name: 🐛 Bug
+description: Report a bug.
+labels: ["status: needs triage", "type: bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **READ BEFORE CREATING!**
+
+  - type: markdown
+    attributes:
+      value: |
+        - Avoid creating duplicates! Read the FAQ page and search through Issues and Discussions before creating one.
```

Comment:
I don't think this is really necessary since there's already a checkbox below.

Reply:
I'll put the checkbox below READ BEFORE CREATING! and the reminders.